services/etcd.go

```diff
@@ -308,6 +309,7 @@ func RunEtcdSnapshotSave(ctx context.Context, etcdHost *hosts.Host, prsMap map[s
 	if err != nil {
 		return err
 	}
+	time.Sleep(EtcdSnapshotWaitTime * time.Second)
```

Comment:
I feel like we might need to add validation for snapshot related parameters prior to starting a container. But at this point, waiting might be the least impactful way in terms of qa

Reply:
I agree, but adding a paremeter validation now will impact the way this feature has been tested, also if we decides that we shouldn't exit on snapshot error then it won't make any difference